<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>853</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>GraPra 2015 - Tycho Brahe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(128, 128, 128, 255));</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="GLWidget" name="main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Optionen</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="BeleuchtungSonne">
         <property name="text">
          <string>Beleuchtung (Sonne)</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Gitternetz">
         <property name="text">
          <string>Gitternetz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxLocalRotation">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Lokale Rotation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="GlobaleRotation">
         <property name="text">
          <string>Globale Rotation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="koordinatSystem">
         <property name="text">
          <string>Koord. System</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Umlaufbahn">
         <property name="text">
          <string>Umlaufbahnen</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Animation: 1.0x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>14</number>
          </property>
          <item>
           <widget class="QSlider" name="sliderAnimationSpeed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Todesstern</string>
         </property>
         <widget class="QSlider" name="angle">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
         <widget class="QCheckBox" name="laserActve">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>23</y>
            <width>92</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>aktivieren</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Vorschau</string>
          </property>
         </widget>
         <widget class="QSlider" name="laser_length">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>63</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Pfad</string>
         </property>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>30</y>
            <width>92</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>aktivieren</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>60</y>
            <width>141</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Merkur</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mars</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jupiter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Saturn</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Uranus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Moon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sun</string>
           </property>
          </item>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Auflösung</string>
         </property>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>83</x>
            <y>23</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="geometry">
           <rect>
            <x>83</x>
            <y>44</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_3">
          <property name="geometry">
           <rect>
            <x>83</x>
            <y>63</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>24</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>U:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>44</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>V:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>63</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Ring</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetCamera">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="text">
          <string>Reset Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>gui/glwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
